src/thunderbots/software/ai/world/field.h

```diff
@@ -128,6 +128,12 @@ class Field
      */
     Rectangle enemyDefenseArea() const;
 
+    /**
+     * Gets the playable area of the field as a rectangle
```

Comment:
What's the playable area? The area just in the field lines? Or the entire area robots can move (which includes a bit of space outside the field lines)? It's not immediately obvious from the function name, and should be part of the javadoc comment.

Reply:
renamed to `fieldLines()` after offline discussion